xml/xml.php

```diff
@@ -382,7 +383,20 @@ function xml_parse_into_struct (#[LanguageLevelTypeAware(["8.0" => "XmlParser"],
  * section.
  */
 #[Pure]
-function xml_get_error_code (#[LanguageLevelTypeAware(["8.0" => "XmlParser"], default: "resource")] $parser): int {}
+function xml_get_error_code ($parser): int|false {}
+
+#[PhpStormStubsElementAvailable('8.0')]
+/**
+ * Get XML parser error code
+ * @link https://php.net/manual/en/function.xml-get-error-code.php
+ * @param XmlParser $parser <p>
+ * A reference to the XML parser to get error code from.
+ * </p>
+ * @return int Returns one of the error codes listed in the error codes
+ * section.
+ */
+#[Pure]
+function xml_get_error_code (XmlParser $parser): int {}
```

Comment:
What do you think about reusing `LanguageLevelTypeAware` here on argument and function to avoid duplication?

Reply:
The parameter, the return and the description are different, so it feels like something different. But I don't know how you want to handle that in general, is there a general rule for this?

Reply:
It's possible to add two `LanguageLevelTypeAware` attributes: for return and for parameter, like `
#[LanguageLevelTypeAware(["8.0" => "int"], default: "int|false")]
function xml_get_error_code (#[LanguageLevelTypeAware(["8.0" => "XmlParser"], default: "resource")] $parser): int|false {}
`
and update description a bit with a note regarding PHP 8